flanker/mime/message/scanner.py

```diff
@@ -28,7 +28,7 @@ def scan(string):
         raise DecodingError("Malformed MIME message"), None, sys.exc_info()[2]
 
 
-def traverse(pointer, iterator, parent=None):
+def traverse(pointer, iterator, parent=None, ignore_report_rfc=False):
```

Comment:
I would suggest renaming it to `allow_bad_mime`, there can be all sorts of incorrect messages in NDNs.